privacy/privacy-shield.md

```diff
@@ -58,18 +58,18 @@ In order to improve our services and the website, and provide more convenient, r
 ## Third Parties
 You understand that Basecamp uses third party vendors and hosting partners to provide the necessary hardware, software, networking, storage, and related technology required to provide you with our services. A current list of vendors is available upon request.
 
-In cases of onward transfer to these third parties for data of EU or Swiss individuals received pursuant to the Privacy Shield Frameworks, Basecamp is potentially liable should any issues or concerns arise.
+In cases of onward transfer to these third parties for data of EU, United Kingdom or Swiss individuals received pursuant to the Privacy Shield Frameworks, Basecamp is potentially liable should any issues or concerns arise.
```

Comment:
Preference on clarity/ambiguity without Oxford comma?

Reply:
```suggestion
In cases of onward transfer to these third parties for data of EU, United Kingdom, or Swiss individuals received pursuant to the Privacy Shield Frameworks, Basecamp is potentially liable should any issues or concerns arise.
```
Yep, that's better 👍🏼